app/interactors/users/checkins/import_checkins.rb

```diff
@@ -14,6 +15,10 @@ def call
 
     private
 
+    def creat_activity
+      device.create_activity :import, owner: device.user, parameters: { count: CSV.read(path).length }
```

Comment:
Kind of, won't that essentially be the same thing as what I'm doing already though? I will still need the create_activity method in the interactors, they will just then call this entity.public_send method? Or am I not understanding you correctly?